plugins/services/src/js/columns/ServicesTableDiskColumn.tsx

```diff
@@ -8,16 +8,17 @@ import Units from "#SRC/js/utils/Units";
 import { SortDirection } from "plugins/services/src/js/types/SortDirection";
 import ServiceTableUtil from "../utils/ServiceTableUtil";
 
+export const ServiceDisk = React.memo(({ resource }: { resource: string }) => (
+  <NumberCell>
+    <span>{Units.formatResource("disk", resource)}</span>
+  </NumberCell>
+));
+ServiceDisk.displayName = "ServiceDisk";
```

Comment:
some components have been lucky while others have been not? or is there a reason, e.g. `ServiceName` does not have/need that property? (it's totally optional from my POV, but still curious)

Reply:
I'll remove them, I only added them to debug another problem but they weren't of any help there.